.devcontainer.json

```diff
@@ -31,7 +31,6 @@
     "ghcr.io/devcontainers/features/powershell:1": {
       "modules": "Az.Resources, Microsoft.Graph.Authentication, Microsoft.Graph.Applications"
     },
-    "ghcr.io/rchaganti/vsc-devcontainer-features/azurebicep:1": {
-    }
+    "ghcr.io/rchaganti/vsc-devcontainer-features/azurebicep:1": {}
```

Comment:
Is this the linter making this change? I just don't want someone else's settings reverting this back again! :-) Feel free to resolve this comment if it's the linter!

Reply:
Yes, it is linter. 
eslint and prettier are now part of package.json